docs/user_guide/trace.md

```diff
@@ -30,13 +30,58 @@
 
 Triton includes that capability to generate a detailed trace for
 individual inference requests. Tracing is enable by command-line
-arguments when running the tritonserver executable. For example,
+arguments when running the tritonserver executable.
+
+`--trace-config` command line option in Triton can be used to specify 
+global and trace mode specific config setting.. The format of this flag 
+is `--trace-config=<mode>,<setting>=<value>`, where `<mode>` 
+is either `triton` or `opentelemetry`. By default, the trace mode is set to `triton`,
+and the server will use Triton's trace APIs. For `opentelemetry` mode, 
+the server will use the [OpenTelemetry's APIs](#opentelemetry-trace-support) to generate, 
+collect and export traces for individual inference requests.
+
+To specify global trace settings (level, rate, count, or mode), 
+the format is `--trace-config=<setting>=<value>`.
+
+An example usage, which invokes Triton's trace APIs:
 
 ```
-$ tritonserver --trace-file=/tmp/trace.json --trace-rate=100 \
-    --trace-level=TIMESTAMPS --trace-log-frequency=50 --trace-count=100 ...
+$ tritonserver \
+    --trace-config=triton,file=/tmp/trace.json \
+    --trace-config=triton,log-frequency=50 \
+    --trace-config=rate=100 \
+    --trace-config=level=TIMESTAMPS \
+    --trace-config=count=100 ...
 ```
 
+## Trace Settings
+### Global Settings
+The following table shows available global trace settings to pass to `--trace-config`
```

Comment:
Having all the flags as part of --trace-config makes sense as even though global, they are configuration of tracer.
I don't think the first equal sign is required. It can be interpreted correctly even with
`--trace-config rate=100`

Reply:
We can probably update the documentation of the cli examples below.

Reply:
I think it make sense to move to one flag for same reasons as @tanmayv25 mentioned. With back to back `=` not required, it became eye-pleasant, and by eliminating separate global flags, users don't need to remember different flag names just to specify trace configuration.

It is also easier from user's side to check how many flags is set. If they have a large number of flags specified, by searching for `--trace-config` they can highlight all trace-related flags and count them for example

Reply:
I think it should be fine to use `--trace-config`.